samples/nrf9160/lwm2m_client/prj.conf

```diff
@@ -101,4 +130,18 @@ CONFIG_ENTROPY_CC3XX=n
 CONFIG_TEST_RANDOM_GENERATOR=y
 
 # Set LwM2M Server IP address here
-CONFIG_APP_LWM2M_SERVER=""
+CONFIG_APP_LWM2M_SERVER="leshan.eclipseprojects.io"
+
+# Set PSK here
+CONFIG_APP_PSK="6E7266736563726574"
+
+# Enable modem trace
+CONFIG_NRF_MODEM_LIB_TRACE_ENABLED=y
```

Comment:
This probably doesn't need to be enabled by default

Reply:
Agreed.

Reply:
Actually, removing this seems to introduce some errors:
`
[00:00:05.874,298] <err> modem_info_params: Link data not obtained: 20 -5
[00:00:05.874,328] <err> modem_info_params: Network data not obtained: -5
[00:00:05.874,328] <err> lwm2m_connmon: Unable to obtain modem parameters: -11
`
It doesn't seem like the error affects anything else, the sample works fine otherwise. What should we do?

Reply:
Ok, the traces are needed then and should be left as enabled.

Reply:
Reenabled the setting.

Reply:
Actually, what I observe is that leaving this config enabled messes up the log output (which makes sense since the traces are likely interfereing with the acutal logs from the logger). So IMO it just covers the issue, not fixes it. I was able to observe the above errors with the traces left enabled (although not always since the log ouput was messed up (incomplete)):
```
[00:00:00.300,872] <inf> lwm2m_firmware: Firmware updated successfully
[00:00:02.310,363] <inf> app_lwmcConnecting toE network.
[00:00:02.310,394] <inf> app_lwm2m_client: This may take vnutes.32m[00:00:08.236,389] <inf> app_lwm2m_clim
[00:00:08.241,638] net_l2m_rient:art LWM2M Client: nrf-32656162687
[00:00:8,53[1;31m<err> modem_inarams: Link d notained: 20 -5
[00:0:08.278,51;31m<err> modem_info_prams: Netwota nobtai -5
uar[[00:00:08.278,564] <err> lwm2m_connmon: Unable to obtain modem parameters: -11;32mu:~$ [8D[00749,481 0
[00:00:10.731,781] <dbg> app_lwm2m_client.rd_client_event: Registration complete
```
This should be sorted out.

Reply:
Turns out that the errors were caused by `CONFIG_MODEM_INFO_ADD_DATE_TIME` and hidden by enabling the traces on Thingy91. I disabled both and it seems to have fixed the issue. I don't understand why enabling the traces messes up the logging on the DK and not on the Thingy though.